### PR TITLE
making a basic test for a webauthn object from a chrome extension

### PR DESCRIPTION
**Pull request by yaleman**
Refers to #288

- [x] cargo test has been run and passes
- [ ] documentation has been updated with relevant examples (if relevant)

### CODE REVIEW

File: webauthn-rs-core/src/core.rs [hunk 1]
```diff
@@ -50,6 +50,10 @@ use base64urlsafedata::Base64UrlSafeData;
 /// as much as possible.
 #[derive(Debug)]
 pub struct WebauthnCore {
+    /// The RP name
+    #[cfg(debug_assertions)]
+    pub rp_name: String,
```
Comment (Firstyear) [inline webauthn-rs-core/src/core.rs:55]: Could just be worth making a getter instead of the debug assertions.

File: webauthn-rs/src/lib.rs [hunk 2]
```diff
@@ -1225,3 +1225,16 @@ impl Webauthn {
         self.core.authenticate_credential(reg, &state.ast)
     }
 }
+
+#[test]
+/// Test building a webauthn object from a chrome extension origin
+fn test_webauthnbuilder_chrome_url() {
```
Comment (Firstyear) [inline webauthn-rs/src/lib.rs:1231]: does the test pass or fail? 🤔

File: webauthn-rs-core/src/core.rs (hunk 1 above)
Comment (yaleman) [reply webauthn-rs-core/src/core.rs:55]: done!

File: webauthn-rs/src/lib.rs (hunk 2 above)
Comment (yaleman) [reply webauthn-rs/src/lib.rs:1231]: it passes! and now the comment mentions it.

**Review by Firstyear — APPROVED** [review]

**Pull request merged**